vaadin-flow-components-shared-parent/vaadin-flow-components-test-util/src/main/java/com/vaadin/tests/AbstractValidationTest.java

```diff
@@ -1,16 +0,0 @@
-package com.vaadin.tests;
-
-import org.junit.BeforeClass;
-
-public abstract class AbstractValidationTest
```

Comment:
**note:** I assumed that this class can be removed likewise since it is no longer used.